How-we-use-pull-requests.md

```diff
@@ -0,0 +1,68 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
```

Comment:
Not sure about the title of this page, this dock doesn't talk about the uses of the PRs, right?

Reply:
I've renamed it to "Pull requests at Oppia"